Examples/test-suite/sizeof_pointer.i

```diff
@@ -1,9 +1,12 @@
-/* 
+/*
 This testcase tests whether the sizeof operator on a pointer is working.
 */
 
 %module sizeof_pointer
 
+#ifdef SWIGFORTRAN
+%warnfilter(SWIGWARN_LANG_IDENTIFIER); /* incompatible array expressions */
```

Comment:
This should be removed if the default array handling works the same as the other languages (under discussion).

Reply:
This is fixed and no longer needed.